### PR TITLE
Some tidy up of the DB update script.

### PR DESCRIPTION
**Pull request by lostcarpark**
Some minor changes to the database update script.

- Fixed a warning when run under PHP 7 about priority of operators (added parentheses).
- Some changes to comply with phpcs checks, mostly around comments and line lengths.
- Added types to function specifications.
- Added messages to indicate how many patches applied, or if none, to say none found.

### CODE REVIEW

File: scripts/db_update.php [hunk 1]
```diff
@@ -76,33 +87,44 @@ function executeSQL($file)
...
 /**
  * Return true if specified table exists in database.
- * @param string $tableName
+ *
+ * @param string $tableName - The name of the table to check presence of.
+ *
  * @return bool
  */
-function checkTableExists($tableName)
+function checkTableExists(string $tableName): bool
```
Comment (bcholmes) [inline scripts/db_update.php:104]: I thought this didn't work in PHP < 8?

Comment (lostcarpark) [reply scripts/db_update.php:104]: Typed parameters and return values are definitely supported in PHP 7.
There are some special cases like union types (e.g. string || null $param), which require PHP8.
I have carried out tests of this change in PHP 7.4 and 8.1.

Comment (bcholmes) [reply scripts/db_update.php:104]: Okay. Good to know.

Comment (lostcarpark) [reply scripts/db_update.php:104]: Sorry, I had the union types slightly wrong. They are actually `string|null $param`. I haven't used any in this script, but I did try to sneak one in when I added the ConfigurationAdmin page.

Comment (bcholmes) [reply scripts/db_update.php:104]: You are a sneaky sneaker who sneaks!

**Pull request merged**